### PR TITLE
fix: added emoji mart lib to personal chats

### PR DESCRIPTION
**Pull request by The-Vector-007**
Screenshot 2022-02-02 220156

### CODE REVIEW

**Comment by The-Vector-007** [comment]
Screenshot 2022-02-03 142744

**Comment by Ashwin-op** [comment]
@The-Vector-007 Not sure if we need to refactor the input component as a whole, but the UI looks different from the ones in channels, labs, etc.

**Comment by The-Vector-007** [comment]
Screenshot 2022-02-08 181523

File: projects/shared-components/discussion-personal-chat/discussion-personal-chat.component.scss [hunk 1]
```diff
@@ -1,8 +1,9 @@
+@import './projects/commudle-admin/src/assets/styles/variables';
```
Comment (Ashwin-op) [inline projects/shared-components/discussion-personal-chat/discussion-personal-chat.component.scss:1]: Do we need this import here?

File: projects/shared-components/discussion-personal-chat/discussion-personal-chat.component.ts [hunk 2]
```diff
@@ -110,141 +110,125 @@ export class DiscussionPersonalChatComponent implements OnInit, OnDestroy {
...
   }
 
+  toggleEmojiForm() {
+    this.showEmojiForm = !this.showEmojiForm;
+  }
+
+  selectEmoji(event) {
+    let currentValue = this.chatMessageForm.get('content').value || '';
+    this.chatMessageForm.patchValue({
+      content: currentValue.concat(`${event.emoji.native}  `),
```
Comment (Ashwin-op) [inline projects/shared-components/discussion-personal-chat/discussion-personal-chat.component.ts:230]: Let's remove the space in the content

**Review by Ashwin-op — CHANGES_REQUESTED** [review]

**Comment by Ashwin-op** [comment]
Couple of more things

- There's a console error, please check it out
image
- Reply message input doesn't have emoji
- Emoji should be inserted wherever the cursor is (this is a bug at every place we are using emoji picker)
- The input should be sticky at the bottom
image

**Review by Ashwin-op — CHANGES_REQUESTED** [review]
Couple of more changes are given as a comment

**Pull request merged**